src/components/cards/GradientCard/GradientCard.vue

```diff
@@ -0,0 +1,33 @@
+<script setup lang="ts">
+/**
+ * TYPES
+ */
+type Props = {
+  imgUrls: string[];
+};
+
+/**
+ * PROPS
+ */
+const props = defineProps<Props>();
+
+const bgImageStyles = computed(() => {
+  const bgStyle = props.imgUrls
+    .map(imgUrl => `bg-[url('${imgUrl}')]`)
```

Comment:
This doesn't work in production builds because tailwind doesn't know the class is being used and removes it. See the preview URL.